community/server/src/main/java/org/neo4j/server/web/Jetty9WebServer.java

```diff
@@ -375,6 +351,20 @@ protected void startJetty() throws Exception
         }
     }
 
+    @Override
+    public InetSocketAddress getLocalHttpAddress()
+    {
+        return new InetSocketAddress( serverConnector.getHost(), serverConnector.getLocalPort() );
```

Comment:
Maybe extract a method like `InetSocketAddress toSocketAddress(ServerConnector)` to use here and in `#getLocalHttpsAddress()`?